lib/node_modules/@stdlib/regexp/to-json/README.md

```diff
@@ -0,0 +1,170 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2022 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# toJSON
+
+> Return a [JSON][json] representation of a [regular expression][regexp] object.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var toJSON = require( '@stdlib/regexp/to-json' );
+```
+
+#### toJSON( regexp )
+
+Returns a [JSON][json] representation of a [regular expression][regexp] object.
+
+```javascript
+var regex = new RegExp('ab+c');
+
+var json = toJSON( regex );
+/* returns
+    {
+        'type': 'regexp',
```

Comment:
One quick change: from looking at some other `to-json` packages, we capitalize the type. For example, `@stdlib/array/to-json` when serializing typed arrays. In which case, @steff456 would you mind changing to `RegExp`?

Reply:
No problem!